src/Algebra/Graph/Relation/InternalDerived.hs

```diff
@@ -111,7 +79,7 @@ instance Ord a => Eq (TransitiveRelation a) where
 instance (Ord a, Show a) => Show (TransitiveRelation a) where
     show = show . transitiveClosure . fromTransitive
 
--- TODO: To be derived automatically using GeneralizedNewtypeDeriving in GHC 8.2
+-- TODO: Optimise the implementation by caching the results of transitive closure.
```

Comment:
This module still contains a lot of accidental changes unrelated to symmetric relations.

Could you please undo **all** unrelated changes?

Reply:
Yes, I'm going to check with the current versions and make sure I dont forget anything